docs/book/user-guide/starter-guide/track-ml-models.md

```diff
@@ -141,13 +141,13 @@ def svc_trainer(
 )
 def training_pipeline(gamma: float = 0.002):
     # Now this pipeline will have the production `iris_classifier` model active.
-    model_version = get_pipeline_context().model_version
+    model = get_pipeline_context().model
 
     X_train, X_test, y_train, y_test = training_data_loader()
     svc_trainer(gamma=gamma, X_train=X_train, y_train=y_train)
 ```
 
-## Logging metadata to the `ModelVersion` object
+## Logging metadata to the `Model` object
 
 [Just as one can associate metadata with artifacts](manage-artifacts.md#logging-metadata-for-an-artifact), model versions too can take a dictionary
```

Comment:
same here as above. we're implicitly mentioning versions without talking about the distinction. I guess it's ok?